packages/terra-notification-dialog/src/terra-dev-site/doc/example/HazardHighVariant.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import Button from 'terra-button';
+import NotificationDialog from 'terra-notification-dialog';
+
+const NotificationDialogExample = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleCloseModal = (actionClicked) => {
+    alert(`You clicked the ${actionClicked} action.`); // eslint-disable-line no-alert
```

Comment:
Using alerts in dev-site examples is a pet-peeve of mine. Can we just close the dialog? I don't know if the examples need to demonstrate which callback is executed, seems like tests would cover that.

Reply:
I agree with the pet peeve. I vote we add a p tag to the dom stating what would have been said in the alert.

Reply:
removed throughout in this comment: https://github.com/cerner/terra-framework/pull/1105/commits/440c28411da12f4c7b6d14c932c0437e8a289e63#diff-cf4f3b43a81f8e1ceb70734455bd7672L10